packages/loader/src/GLTFLoader.ts

```diff
@@ -1,9 +1,29 @@
-import { AssetPromise, AssetType, Loader, LoadItem, resourceLoader, ResourceManager } from "@galacean/engine-core";
+import {
+  AssetPromise,
+  AssetType,
+  Engine,
+  EngineConfiguration,
+  Loader,
+  LoadItem,
+  resourceLoader,
+  ResourceManager
+} from "@galacean/engine-core";
 import { GLTFResource } from "./gltf/GLTFResource";
 import { GLTFParserContext } from "./gltf/parser";
+import { MeshoptDecoder } from "./gltf/extensions/MeshoptDecoder";
 
 @resourceLoader(AssetType.GLTF, ["gltf", "glb"])
 export class GLTFLoader extends Loader<GLTFResource> {
+  static destroyMeshOptWorkers() {
```

Comment:
- Add comments
- Let us unify the mechanism and expression of worker release by GLTFLoader and KTX2Loader as much as possible